sn_node/src/put_validation.rs

```diff
@@ -299,7 +302,7 @@ impl Node {
             None => {
                 // Notify replication_fetcher to mark the attempt as completed.
                 self.network.notify_fetch_completed(key.clone());
-                return Ok(CmdOk::DataAlreadyPresent);
+                return Ok(());
```

Comment:
The trace log to say that the register is already stored, is printed by the `register_validation` function here.